blocks/providers/providers.css

```diff
@@ -0,0 +1,81 @@
+.providers{
+    padding: 0 10px;
+}
+.provider__card{
+    width: 100%;
+    
+}
+.provider__image{
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    height: 326px;
+    position: relative;   
+    border-radius: 20px;
+}
+.provider__image_type_freelance{
+    background-image: url(../../images/freelancers.png);
+}
+.provider__image_type_business{
+    background-image: url(../../images/salonSpa.png);
+}
+.provider__card-content{
+    height: 100%;
+    width: 100%;
+    background-color: rgba(0, 0, 0, .5);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    border-radius: 20px;
+}
+.provider__card-heading{
+    color: #fff;
+    text-align: center;    
+}
+.provider__card-button{
```

Comment:
Override and remove the default border styling, I'm seeing some raised styling. I'm not seeing the right fonts here but that might just be on my end, thought it'd be worth mentioning just in case. Cursor styles are needed here too.

Reply:
Realized I forgot to add things on the fonts, so that was definitely me.
I use firefox, and didn't see a border on my end and forgot about it. But updated!

Reply:
Still need a lil pointer cursor here